src/upgrade/nc_upgrade_manager.js

```diff
@@ -251,7 +251,7 @@ class NCUpgradeManager {
      */
     async _run_nc_upgrade_scripts(this_upgrade) {
         try {
-            await run_upgrade_scripts(this_upgrade, this.upgrade_scripts_dir, { dbg });
+            await run_upgrade_scripts(this_upgrade, this.upgrade_scripts_dir, { dbg, from_version: this_upgrade.package_from_version });
```

Comment:
Could you update the JSDoc of `run_upgrade_scripts` to explain the accepted `options`?